civis_jupyter_notebooks/tests/test_git_utils.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+import os
+import logging
+import six
+import pkg_resources
+import platform
+
+from git.exc import GitCommandError
+from git import Repo
+from civis_jupyter_notebooks.git_utils import CivisGit, GitError
+
+if (six.PY2 or pkg_resources.parse_version('.'.join(platform.python_version_tuple()[0:2]))
+        == pkg_resources.parse_version('3.4')):
+    from mock import patch
+    from mock import MagicMock
+else:
+    from unittest.mock import patch
+    from unittest.mock import MagicMock
```

Comment:
We could probably move this into either a separate module or a pytest fixture file. But also shrug.